Packs/TaniumThreatResponse/ReleaseNotes/2_0_3.md

```diff
@@ -1,3 +1,3 @@
 #### Integrations
 ##### Tanium Threat Response
-- Updated the Docker image to: *demisto/python3:3.9.8.24399*.
+- Updated the Docker image to: *demisto/python3:3.9.8.24399*.
```

Comment:
had to rebase it against the master as the docker image was updated, not sure why git recognizes it as a change as it is basically the same line